pkg/azureclients/vmclient/azure_vmclient.go

```diff
@@ -42,6 +43,7 @@ var _ Interface = &Client{}
 type Client struct {
 	armClient      armclient.Interface
 	subscriptionID string
+	cloudName      string
```

Comment:
this is not used?

Reply:
It is used on the new unit test and in other clients that use the compute API version (interface client for example). As mentioned on my reply to @andyzhangx above, I added cloudName to all clients for consistency.

Reply:
thanks for clarification, then it makes sense